app/views/doctors/search_root.html.haml

```diff
@@ -0,0 +1,4 @@
+%header.site-header
+  %h1.site-title= t(:app_purpose)
+  = render "doctors/doctor_search_form"
+  = button_tag t("get_location.submit_text"), type: "submit", name: nil, id:'get_location'
```

Comment:
Since this button only works if you have JS, we should probably only show it if you do (and if you're device has location capabilities, but that's probably a bit more complex).

I recon the best way to do that is add the button into the form with JS.

What you recon @JasonThomasData ?

Also it shouldn't have the type "submit" cause it doesn't submit anything.